tests/utils.go

```diff
@@ -1633,24 +1633,11 @@ func RunVMIAndExpectLaunchIgnoreWarnings(vmi *v1.VirtualMachineInstance, timeout
 }
 
 func RunVMIAndExpectLaunchWithIgnoreWarningArg(vmi *v1.VirtualMachineInstance, timeout int, ignoreWarnings bool) *v1.VirtualMachineInstance {
-	By(StartingVMInstance)
-	var obj *v1.VirtualMachineInstance
-	var err error
-	virtClient, err := kubecli.GetKubevirtClient()
-	Expect(err).ToNot(HaveOccurred())
-	Eventually(func() error {
-		obj, err = virtClient.VirtualMachineInstance(util2.NamespaceTestDefault).Create(vmi)
-		return err
-	}, timeout, 1*time.Second).ShouldNot(HaveOccurred())
-	By("Waiting until the VirtualMachineInstance starts")
 	if ignoreWarnings {
-		WaitForSuccessfulVMIStartWithTimeoutIgnoreWarnings(obj, timeout)
+		return RunVMIAndExpectLaunchIgnoreWarnings(vmi, timeout)
 	} else {
-		WaitForSuccessfulVMIStartWithTimeout(obj, timeout)
+		return RunVMIAndExpectLaunch(vmi, timeout)
 	}
-	vmi, err = virtClient.VirtualMachineInstance(util2.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
```

Comment:
By removing this a logic change is introduced. The returned is and old instance initialized before the waiting for vmi to start.
I remember discussing it on another PR and solving it by changing the waiting method to return the updated vni instance instead if the node name.

Reply:
Right, please rebase.

Reply:
May I ask why? tide rebases automatically to run all lanes, it tells me if a manual rebases is required.

Reply:
It will help me to see the full picture without trying to merge the PRs in my head. But but not critical, approving.

Reply:
thanks!